modules/edge-machine-pools-aws-local-zones.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: CONCEPT
+[id="edge-machine-pools-aws-local-zones_{context}"]
+= Edge compute pools and AWS Local Zones
+
+Edge worker nodes are tainted worker nodes that run in AWS Local Zones locations.
+
+When deploying a cluster that uses Local Zones:
+
+* Amazon EC2 instances in the Local Zones are more expensive than Amazon EC2 instances in the Availability Zones
+* Latency between applications and end users are lower in Local Zones, and it may vary by location. There is a latency impact for some workloads if, for example, routers are mixed between Local Zones and Availability Zones.
+* The cluster-network Maximum Transmission Unit (MTU) is adjusted automatically to the lower restricted by AWS when Local Zone subnets are detected on the `install-config.yaml`, according to the network plugin.
```

Comment:
```suggestion
* When Local Zone subnets are present in the `install-config.yaml` file, the installer adjusts the MTU based on the network plugin that is used in the cluster. The adjusted MTU value is determined by AWS requirements. Typically, it is between 1200 and 1300.
```

@mtulio Are there specific figures we could give? That would really make this easier to write. :) 

Peeping at the code for a sec, I'm only seeing 1200 and 1250. Is there more to it than that?

Reply:
> Typically, it is between 1200 and 1300.

I think it's not totally safe to say that in a generic way as we require to be 1200 (OVN-K), 1250 (O-SDN), or even lower depending on the features enabled (ipsec, described on the `Important` section).

If you would like to be more direct, maybe you can replace `Typically`  by `For example, the adjusted values are 1200 for OVN-Kubernetes, and 1250 for OpenShift SDN network plugins, needing manual adjustment if more features is enabled.`

Check the 4.12 docs for more information, on where we needed to the user to patch those values manually:

https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-localzone.html#installation-localzone-generate-k8s-manifest_installing-aws-localzone

Reply:
```suggestion
* The cluster-network Maximum Transmission Unit (MTU) is adjusted automatically to the lower restricted by AWS when Local Zone subnets are detected on the `install-config.yaml`, according to the network plugin. For example, the adjusted values are 1200 for OVN-Kubernetes and 1250 for OpenShift SDN. If additional features are enabled, manual MTU adjustment can be necessary.
```